slack_bolt/slack_function/slack_function.py

```diff
@@ -78,8 +77,95 @@ def __call__(*args, **kwargs):
 
         return __call__
 
-    # TODO add view listener
+        # -------------------------
+
+    # view
+
+    def view(
+        self,
+        constraints: Union[str, Pattern, Dict[str, Union[str, Pattern]]],
+        matchers: Optional[Sequence[Callable[..., bool]]] = None,
+        middleware: Optional[Sequence[Union[Callable, Middleware]]] = None,
+    ) -> Callable[..., Optional[Callable[..., Optional[BoltResponse]]]]:
+        """Registers a new `view_submission`/`view_closed` event listener for a function.
+
+            # Use this method as a decorator
+            @app.function("function_1")
+            def sample_view(event, complete: Complete):
+                client.views_open(
+                    interactivity_pointer=interactivity_pointer,
```

Comment:
Do you think we need to explain in here where a user can get an interactivity pointer from?

Reply:
I feel like this should be written in the docs and we should add a reference to it here (but I don't think the docs are written for this yet) I've had some average experiences dealing with very lengthy suggestion boxes in IDEs

Reply:
hm, I think the code snippet here should show a bit more realistic code. So, demonstrating how to extract interactivity_pointer from event data would be preferred.

Reply:
I've added how to extract interactivity_pointer 💯